src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -174,6 +177,11 @@ export class CodeActionModel extends Disposable {
 		this.setState(CodeActionsState.Empty, true);
 	}
 
+	private _showInlineQuickfxies(): boolean {
```

Comment:
[nit] Spelling on `fixes`. Also maybe give this a name that tells it is reading a setting. Starting with `show` suggests to me that calling it will show something